.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -0,0 +1,23 @@
+---
+name: 🐞 Bug
+about: Report a bug
+labels: bug
+
+---
+
+<!-- Thanks for your contribution!
+Please fill out this template as good as possible.
+Please don't include any personal or sensitive data.
+-->
+
+### Current Behavior
+<!-- A concise description of what you're experiencing. -->
+
+### Expected Behavior
+<!-- A concise description of what you expected to happen. -->
+
```

Comment:
Open for discussion: Should we add a headline like `### System Information` or something similar? Especially for bugs, I think it is important to know the system where the bug occurs (operating system, versions, etc.).

Reply:
@mhellmeier I could add [here](https://github.com/eclipse-tractusx/.github/pull/19/files#diff-b577c75bfe1a70d8f33d404b7aa81aff373147db26fe6d5a54283c3dea107f83R22) operating system explicitly, other infos, like version, are already part of the list.

Reply:
Would also be fine for me.

Reply:
done